openmdao/solvers/linear/direct.py

```diff
@@ -223,6 +223,26 @@ def _declare_options(self):
 
         self.supports['implicit_components'] = True
 
+    def check_config(self, logger):
```

Comment:
Previously this check was done for DirectSolver, PETScDirectSolver, and both KSP solvers, so we should probably add this check to the KSP solvers too.

Reply:
Oh yeah, I forgot to check for other usages. Fixed.